internal/fields/model.go

```diff
@@ -42,6 +45,14 @@ func (orig *FieldDefinition) Update(fd FieldDefinition) {
 	if fd.External != "" {
 		orig.External = fd.External
 	}
+	if fd.Index != nil {
+		v := *fd.Index
```

Comment:
Do we need to create a copy of `fd.Index`? The `fd FieldDefinition` is passed by value to `Update`.

Reply:
Good point, I will change this.